pyvista/_vtk.py

```diff
@@ -0,0 +1,382 @@
+                                              vtkArrowSource,
+                                              vtkCylinderSource,
+                                              vtkSphereSource,
+                                              vtkPlaneSource,
+                                              vtkLineSource,
+                                              vtkCubeSource,
+                                              vtkConeSource,
+                                              vtkDiskSource,
+                                              vtkRegularPolygonSource,
+                                              vtkLineSource,
+                                              vtkPointSource,
+                                              vtkArrowSource)
+    from vtkmodules.vtkFiltersGeometry import (vtkGeometryFilter,
+                                               vtkStructuredGridGeometryFilter,
+                                               vtkCompositeDataGeometryFilter,
+                                               vtkDataSetSurfaceFilter)
+    from vtkmodules.vtkFiltersExtraction import (vtkExtractEdges,
+                                                 vtkExtractGeometry,
+                                                 vtkExtractGrid,
+                                                 vtkExtractSelection)
+    from vtkmodules.vtkFiltersTexture import (vtkTextureMapToPlane,
+                                              vtkTextureMapToSphere)
+    from vtkmodules.vtkFiltersPoints import (vtkGaussianKernel,
+                                             vtkPointInterpolator)
+    from vtkmodules.vtkFiltersVerdict import (vtkCellSizeFilter,
+                                              vtkCellQuality)
+    from vtkmodules.vtkImagingGeneral import vtkImageGaussianSmooth
+    from vtkmodules.vtkImagingCore import (vtkExtractVOI,
+                                           vtkImageExtractComponents,
+                                           vtkImageDifference,
+                                           vtkRTAnalyticSource)
+    from vtkmodules.vtkFiltersFlowPaths import vtkStreamTracer
+    from vtkmodules.vtkCommonExecutionModel import vtkImageToStructuredGrid
+    from vtkmodules.numpy_interface.dataset_adapter import (VTKObjectWrapper,
+                                                            numpyTovtkDataArray,
+                                                            VTKArray)
+    from vtkmodules.util.numpy_support import (vtk_to_numpy,
+                                               numpy_to_vtk,
+                                               numpy_to_vtkIdTypeArray,
+                                               get_vtk_array_type)
+    from vtkmodules.vtkCommonCore import (buffer_shared,
+                                          vtkAbstractArray,
+                                          vtkWeakReference)
+
+    # lazy import for some of the less used readers
+    def vtkGL2PSExporter():
```

Comment:
Down the road I could see this pattern confusing some future dev who tries `if isinstance(x, vtkGL2PSExporter)` for example (or anything else that assumes `vtkGL2PSExporter` is the `vtkmodules.vtkIOExportGL2PS.vtkGL2PSExporter` class) and it doesn't do what it seems like it should. I would name these something more like `_lazy_vtkGL2PSExporter` or something to make it clear it's not actually the vtk class itself.

Reply:
Agreed, and I felt a little "lazy" about the way I implemented this.  I'll patch this.

Reply:
Oh I totally missed that this is `from vtkmodules` and below it's `from vtk`!

Reply:
I think this comment was meant for the other thread, right?

Reply:
Whoops, yep!

Reply:
Fixed in fb3f18d.  It's not perfect, but these lazy loaders are only used for a handful of readers.